src/Storefront/Resources/views/storefront/utilities/thumbnail.html.twig

```diff
@@ -108,7 +108,7 @@
                 sizes="{{ sizesValues|join(', ') }}"
                 {% endif %}
             {% endif %}
-            {% for key, value in attributes %}{% if value != '' %} {{ key }}="{{ value }}"{% endif %}{% endfor %}
+            {% for key, value in attributes %}{% if value != '' %} {{ key }}="{{ value }}"{% elseif key == 'alt' %} {{ key }}="{{ value }}"{% endif %}{% endfor %}
```

Comment:
```suggestion
            {% for key, value in attributes %}{% if value != '' %} {{ key }}="{{ value }}"{% elseif key == 'alt' %} {{ key }}="{{ value|striptags|e|trim }}"{% endif %}{% endfor %}
```

Reply:
Good point 👍 Shouldn't we do it for the other attributes as well?